src/core/cubic.c

```diff
@@ -83,6 +96,49 @@ QuicConnLogCubic(
         Cubic->WindowLastMax);
 }
 
+void
+CubicCongestionHyStartChangeState(
+    _In_ QUIC_CONGESTION_CONTROL* Cc,
+    _In_ HYSTART_STATE NewHyStartState
+    )
+{
+    QUIC_CONNECTION* Connection = QuicCongestionControlGetConnection(Cc);
+    QUIC_CONGESTION_CONTROL_CUBIC* Cubic = &Cc->Cubic;
+    switch (NewHyStartState) {
+    case HYSTART_FOUND:
+        Cubic->HyStartCongestionWindow = Cubic->CongestionWindow;
+        break;
+    case HYSTART_DONE:
+    case HYSTART_NOT_FOUND:
+        Cubic->CWndSlowStartGrowthDivisor = 1;
+        break;
+    default:
+        CXPLAT_FRE_ASSERT(FALSE);
+    }
+
+    if (Cubic->HyStartState != NewHyStartState) {
+        QuicTraceEvent(
+            ConnHyStartStateChange,
+            "[conn][%p] HyStart: State=%u CongestionWindow=%u SlowStartThreshold=%u",
```

Comment:
```suggestion
            "[conn][%p] HyStart: State=%hhu CongestionWindow=%u SlowStartThreshold=%u",
```
Your ETW event states `State` is a `uint8_t` not `uint32_t`. Which is it?